wren-ui/src/apollo/server/dataSource.ts

```diff
@@ -123,9 +123,18 @@ const dataSource = {
         DataSourceName.MSSQL,
         connectionInfo,
       );
-      const { host, port, database, user, password } =
+      const { host, port, database, user, password, trustServerCertificate } =
         decryptedConnectionInfo as MS_SQL_CONNECTION_INFO;
-      return { host, port, database, user, password };
+      return {
+        host,
+        port,
+        database,
+        user,
+        password,
+        kwargs: {
+          TrustServerCertificate: trustServerCertificate,
+        },
```

Comment:
According to #698, you need to convert it to string `YES`
if false then don't provide the property